docs/getting-started/concepts.md

```diff
@@ -49,3 +49,46 @@ Iter8 is written in `go` and builds on a few awesome open source projects includ
 - [Fortio](https://github.com/fortio/fortio)
 - [ghz](https://ghz.sh)
 - [plotly.js](https://github.com/plotly/plotly.js)
+
+## AutoX
+
+AutoX, short for "auto experimentation", allows Iter8 to detect changes to your Kubernetes resources objects and automatically start new experiments, allowing you to test your applications as soon as you push out a new version.
+
+To configure AutoX, you will need to specify a set of experiment groups and, for each group, the Kubernetes resource object (a.k.a. trigger object) that you expect AutoX to watch and one or more experiments to be performed in response to new versions of this object.
```

Comment:
This detailed explanation can go under a topic called "Automated experiments (autox)".